src/Stripe.net/Services/Terminal/ConnectionTokens/ConnectionTokenService.cs

```diff
@@ -0,0 +1,33 @@
+namespace Stripe.Terminal
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Stripe.Infrastructure;
+
+    public class ConnectionTokenService : Service<ConnectionToken>,
+        ICreatable<ConnectionToken, ConnectionTokenCreateOptions>
+    {
+        public ConnectionTokenService()
+            : base(null)
+        {
+        }
+
+        public ConnectionTokenService(string apiKey)
+            : base(apiKey)
+        {
+        }
+
+        public override string BasePath => "/terminal/connection_tokens";
+
+        public virtual ConnectionToken Create(ConnectionTokenCreateOptions options, RequestOptions requestOptions = null)
```

Comment:
I wonder if we should provide overloads with no options. It seems a little bit silly to force users to create a `ConnectionTokenCreateOptions` instance that contains nothing.

Reply:
I hear you but then it becomes a mess of overloads and I'm not a big fan, we saw what it did to stripe-java.

Reply:
Just tested it, and we can simply set `null` as the default value for the `options` argument. It still satisfies the interface and makes the argument optional without requiring an overload.

Reply:
fixed, PTAL